Filter/StringFilter.php

```diff
@@ -55,7 +55,7 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
      *
      * @return bool
      */
-    private function getOperator($type)
+    protected function getOperator($type)
```

Comment:
IMO it's more of a doctrine bug. Doctrine should use `ILIKE` when I write `LIKE` in DQL.